packages/ember/tests/routing/router_service_test/urlFor_test.js

```diff
@@ -0,0 +1,67 @@
+import { inject } from 'ember-runtime';
+import { Component } from 'ember-glimmer';
+import { Route, NoneLocation } from 'ember-routing';
+import {
+  get,
+  set
+} from 'ember-metal';
+import {
+  RouterTestCase,
+  moduleFor
+} from 'internal-test-helpers';
+
+import { isFeatureEnabled } from 'ember-metal';
+
+if (isFeatureEnabled('ember-routing-router-service')) {
+  moduleFor('Router Service - urlFor', class extends RouterTestCase {
+    constructor() {
```

Comment:
I had been doing some custom work in this constructor, but removed it. The constructor was just left in place. Will remove!